boot/setup.sh

```diff
@@ -134,6 +134,13 @@ function manta_setup_nginx {
 }
 
 
+function manta_setup_rebalancer {
```

Comment:
naming nit: I would call this "manta_setup_rebalancer_agent".

Reply:
Now's a good time to change it - doing so!